### PR TITLE
define migrations skeleton

### PR DESCRIPTION
**Pull request by jahdevelops**

### CODE REVIEW

File: src/controller/seller/order.controller.js
Comment (Lekejosh) [inline src/controller/seller/order.controller.js:1]: image

Fix the errors

File: seeders/20240122220926-demo-flow.js
Comment (Lekejosh) [inline seeders/20240122220926-demo-flow.js:1]: This is not working... Test and fix the errors

**Comment by Lekejosh** [comment]
The new commits still has error expecially with the seeders file and no fix has been made to the order api route?

File: seeders/20240122220926-demo-flow.js
Comment (jahdevelops) [reply seeders/20240122220926-demo-flow.js:1]: This has been fixed. The command has changed from `npm run seeders` to `npm run seed-mysql` [commit](https://github.com/Vend-Dev/vend-backend/pull/5/commits/192c96926ee838f4996b2964780679ec4273e0de)

**Comment by jahdevelops** [comment]
The 

> The new commits still has error expecially with the seeders file and no fix has been made to the order api route?

This order api route had been fixed

File: src/controller/seller/order.controller.js
Comment (jahdevelops) [reply src/controller/seller/order.controller.js:1]: This order api routes has been fixed

Comment (Lekejosh) [reply src/controller/seller/order.controller.js:1]: Did you test this route? It keeps returning empty array after successfully paying an Order

File: seeders/20240122220926-demo-flow.js
Comment (Lekejosh) [reply seeders/20240122220926-demo-flow.js:1]: This fix here should be better than hard coding it and also run `npm rm uuid` to removed the unused package... Also Test this on your end

File: package.json
Comment (Lekejosh) [inline package.json:1]: Test the seed-mysql fix on your end... This also uses the development mode to run

**Review by Lekejosh — CHANGES_REQUESTED** [review]
Check and fix the requested changes

File: seeders/20240122220926-demo-flow.js
Comment (jahdevelops) [reply seeders/20240122220926-demo-flow.js:1]: The hardcoded uuids were just for development/testing the seeded models in integration with the routes, so you can easily copy the uuid from there and test the supposed route

File: src/controller/seller/order.controller.js
Comment (jahdevelops) [reply src/controller/seller/order.controller.js:1]: Changes has been made to the route, ensure `carts` attribute is saved as JSON when writing to the order model.

**Review by Lekejosh — APPROVED** [review]
Closing this pull request, but will still work on the order controller

**Comment by Lekejosh** [comment]
Resolve the conflicts

**Pull request merged**